### PR TITLE
Feature/thumb process

### PR DESCRIPTION
**Pull request by ladeirarodolfo**
It includes
- Processing of thumbnails
     - Thumbnail creation based on imageFrame (recompressed or not)
     - Commands to filter or not use recompressed data
     - Creation at frame level but only (the favorite) one is promoted to studies, series, instance paths (no support for frame path )
     - The favorite thumb is the middle item of all found item on inner path
 - New package name static-cs-lite
     - It can do some cs stuff at server side (it uses JSDOM)
     - Some implementation was done in order to satisfy the api getRenderedBuffer (some came from wadoImageLoader and some other from OHIF)

### CODE REVIEW

File: packages/static-cs-lite/lib/image/createImage.js [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+    image.modalityLUT = modalityLUTSequence[0];
+  }
+
+  // VOI LUT
+  if (voiLUTSequence && voiLUTSequence.length > 0) {
+    image.voiLUT = voiLUTSequence[0];
+  }
+
+  if (image.color) {
+    image.windowWidth = 255;
```
Comment (wayfarer3130) [inline packages/static-cs-lite/lib/image/createImage.js]: These values should be 256 and 128, according to the comment in DICOM part 3:
A Window Center of 2^(n-1) and a Window Width of 2^n selects the range of input values from 0 to 2n-1. This represents a mathematical identity VOI LUT transformation over the possible input values (whether used or not) in the case where no Modality LUT is specified and the stored pixel data are n bit unsigned integers.

File: packages/static-cs-lite/lib/image/createImage.js [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+    image.windowCenter = 127;
+  }
+
+  // set the ww/wc to cover the dynamic range of the image if no values are supplied
+  if (image.windowCenter === undefined || image.windowWidth === undefined) {
+    const maxVoi = image.maxPixelValue * image.slope + image.intercept;
```
Comment (wayfarer3130) [inline packages/static-cs-lite/lib/image/createImage.js:145]: And these values are also offset - 
windowWidth = maxVoi-minVoi + 1
windowCenter = (maxVoi+minVoi + 1)/2
thus for 0..255, we get:
ww = 255-0 + 1 = 256
wc = (255 + 0 + 1) / 2 = 128

File: packages/static-cs-lite/lib/image/createImage.js (hunk 1 above)
Comment (ladeirarodolfo) [reply packages/static-cs-lite/lib/image/createImage.js]: ok, thank you for the input

File: packages/static-cs-lite/lib/image/createImage.js (hunk 2 above)
Comment (ladeirarodolfo) [reply packages/static-cs-lite/lib/image/createImage.js:145]: ok, thank you for the input

**Pull request merged**